NangPaGo-api/NangPaGo-app/src/main/java/com/mars/app/domain/recipe/controller/RecipeController.java

```diff
@@ -56,6 +59,11 @@ public ResponseDto<Integer> getLikeCount(@PathVariable Long id) {
         return ResponseDto.of(recipeLikeService.getLikeCount(id));
     }
 
+    @GetMapping("/{id}/like/stream")
```

Comment:
엔드포인트명을 조금 더 명확하게 짓는게 좋을 것 같아요.

```suggestion
    @GetMapping("/{id}/like/notification/subscribe")
```